### DISCUSSION TITLE
How to disable noice.nvim scroll keybinds?

### DISCUSSION THREAD

**Discussion by lpanebr** [LazyVim/LazyVim#306] [Q&A] [answered]
I have this in my `config/keymaps.lua`

```lua
vim.keymap.set(
  "n",
  "c-f",
  ":exe '!tmux neww tmux-sessionizer'",
  { expr = true, silent = true, desc = "Tmux sessionizer" }
)
```

but `<c-f>` is still mapped to noice.nvim scroll.

I tried adding this to the `plugins/init.lua` but it did did not work.

```lua
return {
  {
    -- Try to disable noice.nvim scroll
    "folke/noice.nvim",
    keys = {
      { "<c-f>", false },
      { "<c-b>", false },
    },
  },
}
```

Any idea, please??

**Comment by folke** [comment, marked as answer]
Add `, mode = {"i", "n", "s"}` to the mappings you disable

**Comment by lpanebr** [comment]
Thanks! It works.

I also had a problem with  my actual command. I had to add a `<cr>` and remove the `expr = true, `.

Finally this works.

```lua
vim.keymap.set(
  "n",
  "<c-f>",
  ":exe '!tmux neww tmux-sessionizer'<cr>",
  { silent = true, desc = "Tmux sessionizer" }
)
```

Thanks!
